coursemapper-kg/recommendation/app/services/course_materials/recommendation/youtube_service.py

```
@@ -123,10 +177,11 @@
 
         return video_data
 
-    def get_video_details(self, video_id):
-        print("get_video_details for id -------------------- ", video_id)
+    def get_video_details(self, youtube_api_sinlge, video_id):
```

Ruff (0.12.2)

Missing return type annotation for public function `get_video_details`

(ANN201)